usr.sbin/kbdcontrol/kbdcontrol.c

```diff
@@ -1058,14 +1053,11 @@ set_keyrates(char *opt)
 	int r, d;
 
 	if (!strcmp(opt, "slow")) {
-		delay = 1000, repeat = 500;
-		d = 3, r = 31;
+		delay = 1000, repeat = 504 , d = 3, r = 31;
```

Comment:
I know the old code uses , operators, but it would be better to avoid that and use normal ; here and below...